singlestoredb/fusion/handlers/job.py

```diff
@@ -0,0 +1,574 @@
+#!/usr/bin/env python3
+import json
+from typing import Any
+from typing import Dict
+from typing import Optional
+
+from .. import result
+from ...management.utils import to_datetime
+from ..handler import SQLHandler
+from ..result import FusionSQLResult
+from .utils import dt_isoformat
+from .utils import get_workspace_manager
+from singlestoredb.management.job import Mode
+
+
+class ScheduleJobHandler(SQLHandler):
+    """
+    SCHEDULE JOB USING NOTEBOOK notebook_path
+        with_mode
+        [ create_snapshot ]
+        [ with_runtime ]
+        [ with_name ]
+        [ with_description ]
+        [ execute_every ]
+        [ start_at ]
+        [ resume_target ]
+        [ with_parameters ]
+    ;
+
+    # Path to notebook file
+    notebook_path = '<notebook-path>'
+
+    # Mode to use (either Once or Recurring)
+    with_mode = WITH MODE '<mode>'
+
+    # Create snapshot
+    create_snapshot = CREATE SNAPSHOT
+
+    # Runtime to use
+    with_runtime = WITH RUNTIME '<runtime-name>'
+
+    # Name of the job
+    with_name = WITH NAME '<job-name>'
+
+    # Description of the job
+    with_description = WITH DESCRIPTION '<job-description>'
+
+    # Execution interval in minutes
+    execute_every = EXECUTE EVERY <integer>
```

Comment:
I think it would be better to allow a unit to be specified (or at the very least make "MINUTES" part of the syntax). Just having a bare number looks a little odd. I think adding `{ MINUTES | HOURS | DAYS }` would be nice.

Reply:
Another alternative would be to require a token than specifies an interval. I think we use that in some other parts of the Management API. Things like `5s` for 5 seconds, `5m` for 5 minutes, `5d` for 5 days.

Reply:
Implemented `{ MINUTES | HOURS | DAYS } `as aligned with @ricardoasmarques